tests/test_webhook_verification.py

```diff
@@ -20,7 +20,7 @@
     "resource_type": "check",
     "object": {
         "id": "check-123",
-        "href": "https://api.onfido.com/v3/checks/check-123",
+        "href": "https://api.onfido.com/v3.1/checks/check-123",
```

Comment:
@Phoebe-B Same comment as above:

I realise in hindsight this should have v3 in the href, since it's the webhook version, not the API version